src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -605,22 +648,62 @@ typedef NS_ENUM(NSUInteger, MTREventPriority) {
 
 @interface MTREventReport : NSObject
 @property (nonatomic, readonly, copy) MTREventPath * path;
+
+/**
+ * eventNumber will not have a useful value if error" is not nil.
```

Comment:
Missing a quotation mark:
```suggestion
 * eventNumber will not have a useful value if "error" is not nil.
```

Reply:
That said, this comment is a double negative and could read awkwardly? Perhaps "eventNumber only contains a valid value when "error" is nil"?